resources/js/models/CampaignSheet.js

```diff
@@ -225,11 +225,15 @@ class CampaignSheet {
 
     fillDefaultCrossGameItems() {
         if (_.isEmpty(this.crossGameItems)) {
-                this.crossGameItems = {
-                'gh': false,
-                'jotl': false,
-                'cs': false,
-                'fh': false,
+            this.crossGameItems = {
+                'gh': {
```

Comment:
One thing I hadn't considered until now was the fact there might be existing sheet data for campaigns that when loaded would cause problems... my preference would be to somehow detect this state and reset based on this design but I figured I would call out anyway.

Reply:
I've added something to address this, but will leave comment open for any feedback since it's probably the most controversial part of this change.